store/account/account.go

```diff
@@ -60,32 +57,64 @@ func NewWithDB(db *gorm.DB) Store {
 	}
 }
 
-func (t *store) InsertERC20(code *model.ERC20) error {
+func (t *store) InsertERC20(code *model.ERC20, onlySubscribe bool) error {
 	// Insert contract code
 	if err := t.db.Create(code).Error; err != nil {
 		return err
 	}
-	// Create a table for this contract
-	if err := t.db.CreateTable(model.ERC20Storage{
-		Address: code.Address,
-	}).Error; err != nil {
-		return err
+	if onlySubscribe {
+		// Create a account table for this contract
+		if err := t.db.CreateTable(model.Account{
+			ContractAddress: code.Address,
+		}).Error; err != nil {
+			return err
+		}
+	} else {
+		// Create a table for this contract
+		if err := t.db.CreateTable(model.ERC20Storage{
+			Address: code.Address,
+		}).Error; err != nil {
+			return err
+		}
 	}
 	// Create erc20 transfer event table
-	if err := t.db.CreateTable(model.ERC20Transfer{
+	if err := t.db.CreateTable(model.Transfer{
 		Address: code.Address,
 	}).Error; err != nil {
 		return err
 	}
 	return nil
 }
 
+func (t *store) FindERC20(address common.Address) (result *model.ERC20, err error) {
+	result = &model.ERC20{}
+	err = t.db.Where("address = ?", address.Bytes()).Limit(1).Find(result).Error
+	return
+}
+
+func (t *store) ListERC20() (results []model.ERC20, err error) {
```

Comment:
To be consistent with return type of `FindERC20 `, return []*model.ERC20 instead of `[]model.ERC20`

Reply:
OK